packages/amplify/amplify_flutter_android/android/src/test/kotlin/com/amazonaws/amplify/AtomicResultTest.kt

```diff
@@ -58,18 +54,4 @@ class AtomicResultTest {
         atomicResult.success(null)
         verify(mockResult, times(1)).success(any())
     }
-
-    @Test
```

Comment:
This test was flaky on both stable/beta, example failure https://github.com/aws-amplify/amplify-flutter/actions/runs/4297577694/jobs/7490721332. Seems to fail half or 1/3 the time. Would love to keep the test if anyone has a suggestion/insight for why this may be that I could investigate further. I wasn't sure how much it was worth investigating.